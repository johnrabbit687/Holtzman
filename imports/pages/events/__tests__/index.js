import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import Event from "../";

describe("Event", () => {
  it("1. renders with props", () => {
    const wrapper = shallow(<Event.Event />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it("2. pulls the route and snapshots it", () => {
    const wrapper = Event.Routes;
    expect(wrapper).toMatchSnapshot();
  });
})
