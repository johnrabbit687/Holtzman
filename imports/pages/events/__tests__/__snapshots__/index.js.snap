exports[`Event 1. renders with props 1`] = `
<div>
  <p>
    This is an event
  </p>
</div>
`;

exports[`Event 2. pulls the route and snapshots it 1`] = `
Array [
  Object {
    "component": [Function],
    "path": "events/:id",
  },
]
`;
